### ISSUE TITLE
Requires-Python >=3.6,<3.9 ??

### ISSUE THREAD

**Issue by ACatAC** [anntzer/mplcursors#56]
When trying to install mplcursor with python 3.9 via pip I get this error:

```
ERROR: Ignored the following versions that require a different python version: 0.52.0 Requires-Python >=3.6,<3.9; 0.52.0rc3 Requires-Python >=3.6,<3.9
ERROR: Could not find a version that satisfies the requirement mplcursor (from versions: none)
ERROR: No matching distribution found for mplcursor
```
Which I do not quite understand in view of this line in setup.py:
[python_requires=">=3.6"](https://github.com/anntzer/mplcursors/blob/27f0928346f6f45c16136e7676272da5f918bb4f/setup.py#L71)

My full requirements.txt here:
```
pandas==1.4.3
numpy==1.22.4
scikit-learn==1.1.2
xgboost==1.6.1
holidays==0.14.2
azure-core==1.25.1
azure-storage-blob==12.13.1
python-dotenv==0.21.0
matplotlib==3.4.3
seaborn
shap
ray
SQLAlchemy==1.4.41
pyodbc==4.0.34
ipython
eli5
shap-hypetune
hyperopt
flaml
mpl_axes_aligner
yellowbrick
numba
ml-insights
imbalanced-learn
chime
statsmodels
mplcursor
```

**Comment by anntzer** [comment]
Your package name is wrong, it's mplcursors, not mplcursor.
